docs/src/apps/jsonrpc-api.md

```diff
@@ -395,12 +396,34 @@ The signatures will be ordered based on the Slot in which they were confirmed in
 
 ```bash
 // Request
-curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc": "2.0","id":1,"method":"getConfirmedSignaturesForAddress","params":["6H94zdiaYfRfPfKjYLjyr2VFBg6JHXygy84r3qhc3NsC", 0, 100]}' localhost:8899
+curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc": "2.0","id":1,"method":"deprecatedGetConfirmedSignaturesForAddress","params":["6H94zdiaYfRfPfKjYLjyr2VFBg6JHXygy84r3qhc3NsC", 0, 100]}' localhost:8899
 
 // Result
 {"jsonrpc":"2.0","result":{["35YGay1Lwjwgxe9zaH6APSHbt9gYQUCtBWTNL3aVwVGn9xTFw2fgds7qK5AL29mP63A9j3rh8KpN1TgSR62XCaby","4bJdGN8Tt2kLWZ3Fa1dpwPSEkXWWTSszPSf1rRVsCwNjxbbUdwTeiWtmi8soA26YmwnKD4aAxNp8ci1Gjpdv4gsr","4LQ14a7BYY27578Uj8LPCaVhSdJGLn9DJqnUJHpy95FMqdKf9acAhUhecPQNjNUy6VoNFUbvwYkPociFSf87cWbG"]},"id":1}
 ```
 
+### getConfirmedSignaturesForAddress
+
+Returns confirmed signatures for transactions involving an
+address backwards in time from the provided slot
+
+#### Parameters:
+* `<string>` - account address as base-58 encoded string
+* `<object>` - (optional) Configuration object containing the following field:
+  * `startAfter: <string>` - start searching backwards from this transaction signature,
+                             which must be a confirmed signature for the account
+                             address.  If not provided the search starts from
```

Comment:
> which must be a confirmed signature for the account address

I think we can relax this requirement to be "any confirmed signature".  This'll be quite powerful, "I want the transactions that affected address X prior to this other rando signature"

Reply:
Although this does require us to impose an order for transactions that are within a ledger entry, where those transactions are really just a set.   The order we impose would just be whatever order they happen to be in within the entry.   This is a little fib, probably fine right?

Reply:
Looks to me that transaction ordering matters for the entry merkle trees which get included in POH so I think it's actually legit!